packages/x-components/static-docs/build-search-ui/README.md

```diff
@@ -20,7 +20,7 @@ Start building your search and discovery experience:
 
 ::: slot right-intro-column
 
-<img :src="$withBase('/assets/media/build-search-ui.svg')" alt="Build Your Search UI">
+<img :src="$withBase('/assets/media/interface/build-search-ui.svg')" alt="Build Your Search UI">
```

Comment:
This is the only img that's still using `$withBase`. Is this intended or did you forget to change it here too?

Reply:
yes! It's OK since here we're using HTML code since it's contained within an html component. It's the only use case where we don't use MD syntax